tools/castle-editor/code/visualizetransform.pas

```diff
@@ -605,6 +617,13 @@ procedure TVisualizeTransform.GizmoHasModifiedParent(Sender: TObject);
     OnParentModified(Self);
 end;
 
+procedure TVisualizeTransform.GizmoStopDrag(Sender: TObject);
+begin
+  if Assigned(OnGizmoStopDrag) then
+    OnGizmoStopDrag(Self);
+end;
+
```

Comment:
Additional newline.

Reply:
fixed in 123ef82e873ac874b687605da5d13e5693b53fd8